.github/workflows/ci.yml

```
@@ -0,0 +1,59 @@
+        with:
+          node-version: 20
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v4 # 최신 버전인 v4 사용
+        with:
+          node-version: ${{ matrix.node-version }} # Node.js 20.x 버전
+          cache: "npm" # npm 캐시 활성화
+
+      # node_modules 캐시 설정
+      - name: Cache node_modules
+        id: node-modules-cache
+        uses: actions/cache@v3 # 최신 버전인 v3 사용
```

actionlint (1.7.4)

the runner of "actions/cache@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)